requirements/constraints.txt

```diff
@@ -15,3 +15,7 @@
 # This should be removed once the issue is fixed with a new astroid release or with a test_generate_avro_schemas.py
 # module refactor.
 astroid<3.0.0
+
+
+# Temporary to Support the python 3.11 Upgrade
+backports.zoneinfo;python_version<"3.9"  # Newer versions have zoneinfo available in the standard library
```

Comment:
[This](https://github.com/openedx/openedx-events/actions/runs/8640194895/job/23687758476) happened while upgrading openedx-events, so I added this line to solve it. I first saw it here: https://github.com/openedx/edx-platform/pull/34374/files#diff-5d9561e5b3b3e81ab39e0475363c7efc72c9c2dfe29940be48ceb72f8a10f173R52-R53